src/NHibernate.Test.VisualBasic/Program.vb

```diff
@@ -1,4 +1,4 @@
-﻿#If NETCOREAPP2_0
+﻿#If NETCOREAPP2_0_OR_GREATER
```

Comment:
nit: I don't like it. Why not simply NETCORE?

Reply:
Because `NETCOREAPP` could mean `NETCOREAPP1_0`. These are standard symbols. I've also decided not to change our custom defined `NETFX` to standard `NETFRAMEWORK` or `NET461_OR_GREATER`

Reply:
Oh I see. Though I still think using simple NETCORE would minimize the frictions in the future..

Reply:
> I still think using simple NETCORE would minimize the frictions in the future..

It's quite opposite. Because we would need to maintain that constant somehow. 

I actually tried doing so. I wanted to be ready before .NET 6 release and that the effort to enable .NET 6 would be as simple as adding `;net60` to NhAppTargetFrameworks constant. So that was needed to be compatible and I had to maintain the constant manually because I did not want to push people into upgrading of the SDK until the .NET 6 release.

The ugliness and burden of this approach is a big part of why I decided to wait until .NET 6 release to propose the changes.